dev-docs/bidders/appnexus.md

```diff
@@ -2,28 +2,44 @@
 layout: bidder
 title: AppNexus
 description: Prebid AppNexus Bidder Adaptor
-
 top_nav_section: dev_docs
 nav_section: reference
-
-hide: true
-
 biddercode: appnexus
-
 biddercode_longer_than_12: false
-
+hide: true
 ---
 
-
 ### bid params
 
 {: .table .table-bordered .table-striped }
-| Name          | Scope    | Description                                                                                                                                 | Example                    |
-| :---          | :----    | :----------                                                                                                                                 | :------                    |
-| `placementId` | required | The placement ID from AppNexus.  You may identify a placement using the `invCode` and `member` instead of a placement ID.                   | `"234234"`                 |
-| `randomKey`   | optional | A key specified by the publisher. The value maps to a querystring segment for enhanced buy-side targeting. Multiple k-v pairs can be added. | `randomKey` => `randomVal` |
-| `invCode`     | optional | The inventory code from AppNexus. Must be used with `member`                                                                                | `"abc123"`                 |
-| `member`      | optional | The member ID  from AppNexus. Must be used with `invCode`                                                                                   | `"12345"`                  |
-| `reserve`     | optional | Sets a floor price for the bid that is returned                                                                                             | `0.90`                     |
+| Name           | Scope    | Description                                                                                                                                                                                        | Example           |
+|----------------+----------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------|
+| `placementId`  | required | The placement ID from AppNexus.  You may identify a placement using the `invCode` and `member` instead of a placement ID.                                                                          | `"234234"`        |
+| `arbitraryKey` | optional | This key can be *any publisher-defined string*. The value (also a string) maps to a querystring segment for enhanced buy-side targeting. Multiple key-value pairs can be added. See example below. | `'genre': 'rock'` |
```

Comment:
maybe remove the `` from  `arbitraryKey` to show its not a specific name?

Reply:
Good idea, I'll do that!